src/components/helper-methods.js

```diff
@@ -58,10 +58,17 @@ export default {
     }, {parent: parentProps});
   },
 
-  getCalculatedValues(props, defaultStyles) {
-    const style = Helpers.getStyles(props.style, defaultStyles, "auto", "100%");
-    const colors = Array.isArray(props.colorScale) ?
-      props.colorScale : Style.getColorScale(props.colorScale);
+  getCalculatedValues(props, fallbackProps) {
+    const styleObject = props.theme && props.theme.pie ? props.theme.pie.style
```

Comment:
define the `theme` variable first and make use of it here to simplify this ternary